### PR TITLE
Rails 6 revamp

### PR DESCRIPTION
**Pull request by maxkadel**

### CODE REVIEW

**Review by kevinreiss — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ruby 2.7.5
+yarn 1.22.18
+nodejs 12.22.12
+npm 6.14.16
```
Comment (christinach) [inline .tool-versions:4]: I dont think we need to specify npm in the tool-versions

File: Gemfile [hunk 2]
```diff
@@ -45,7 +45,7 @@ gem 'omniauth-rails_csrf_protection'
 gem 'net-ldap'
 gem 'pagy', '~> 3.5'
 
-gem 'webpacker', '>= 4.0.x'
+gem 'webpacker', '>= 4.0.x', '~> 5'
```
Comment (christinach) [inline Gemfile:48]: 👍🏻  https://github.com/pulibrary/lockers_and_study_spaces/blob/98f433d9a2cd2d643c025bcbc528ef6ddf8fdf40/package.json#L5

**Review by christinach — COMMENTED** [review]
@maxkadel one question related to the tool-versions file

**Review by christinach — APPROVED** [review]

File: .tool-versions (hunk 1 above)
Comment (maxkadel) [reply .tool-versions]: Ok, removed, thanks!

**Pull request merged**